entities/communication/voiceCall.json

```diff
@@ -0,0 +1,70 @@
+{
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "Unique identifier for the call"
+    },
+    "duration": {
+      "type": "int",
+      "description": "Duration of the call in seconds"
+    },
+    "created_at": {
+      "type": "timestamp",
+      "description": "Time the call was created"
+    },
+    "updated_at": {
+      "type": "timestamp",
+      "description": "Time the call was last updated"
+    },
+    "status": {
+      "type": "enum",
+      "values": [
+        "incoming call",
+        "answered by staff",
+        "missed",
+        "answered by vonage",
```

Comment:
I would not use "vonage", this may change in the future. I think we should use "line provider"

Reply:
But "answered by line provider" is not a valid value for 'status' field, should we still change it?

Reply:
If it's an existing status, than there are 2 options:
1. keep it this way
2. fix it throughout the code (preferable but not always feasible)
I suggest you take it with Eliran, whatever he decides. It's really a bad practice to have a provider name in our code.